src/deephaven/plot/express/data_generator/example_data.py

```diff
@@ -0,0 +1,170 @@
+        # rough model of the distribution of trade sizes in real market data
+        # they bucket into human sized trade blocks
+        size_dist = random.choices(
+            [0, 2, 3, 4, 5, 10, 20, 50, 100, 150, 200, 250, 300, 400, 500, 1000],
+            [1000, 30, 25, 20, 15, 5, 5, 20, 180, 5, 15, 5, 8, 7, 8, 2],
+        )[0]
+        if size_dist == 0:
+            size = math.ceil(1000 * abs_rand)
+            # round half of the numbers above 1000 to the nearest ten
+            return round(size, -1) if (size > 1000 and rand > 0) else size
+        else:
+            return size_dist
+
+    static_table = (
+        empty_table(size_in_seconds)
+        .update(
+            formulas=[
+                "timestamp = base_time + (long)(ii * SECOND / ticks_per_second)",
+                # must be first as it sets python seed
+                "random_double = (double)random_gauss(i)",  # nicer looking starting seed
+                "sym = random_list_sym(i)",
+                "exchange = random_list_exchange(i)",
+                "side = random_double >= 0 ? `buy` : `sell`",
+                "size = random_trade_size(random_double)",
+                "sym_index = list_sym_index(sym)",
+                "index = i",
+            ]
+        )
+        # generate data for price column
+        .update_by(
+            ops=[
+                rolling_sum_tick(
+                    cols=["rolling_sum = random_double"], rev_ticks=2000, fwd_ticks=0
+                )
+            ],
+            by=["sym"],
+        )
+        .update_by(
+            ops=[ema_tick(decay_ticks=2, cols=["ema = rolling_sum"])], by=["sym"]
+        )
+        # generate date for "SPet500" a hypothetical composite index
+        .update_by(
+            ops=[
+                rolling_sum_tick(
+                    cols=["rolling_sum_individual = random_double"],
+                    rev_ticks=1000,
+                    fwd_ticks=0,
+                )
+            ],
+        )
+        .update_by(
+            ops=[
+                ema_tick(
+                    decay_ticks=5, cols=["ema_individual = rolling_sum_individual"]
+                )
+            ]
+        )
+        # spread the staring values based on symbol and round to 2 decimals
+        .update(
+            [
+                "price = Math.round((ema + init_value + Math.pow(1 + sym_index, 3))*100)/100",
+                "SPet500 = ema_individual + init_value + 50",
+                "dollars = size * price",
+            ]
+        )
+        # drop intermediary columns, and order the output nicely
+        .view(
+            [
+                "timestamp",
+                "sym",
+                "exchange",
+                "size",
+                "price",
+                "dollars",
+                "side",
+                "SPet500",
+                "index",
+                "random = random_double",
+            ]
+        )
+    )
+
+    if ticking == True:
```

Comment:
The `== True` is redundant here
```suggestion
    if ticking:
```